blocked-edges/4.10.14-release-data-with-hyphen-prefix.yaml

```diff
@@ -5,11 +5,4 @@ name: ReleaseDataWithHyphenPrefix
 message: |-
   Clusters updating out of 4.10.14 may, depending on their past history, fail to initiate the update to any later release.  In order to avoid this trouble, we recommend avoiding updates to this release.  The linked solution includes steps to avoid the issue if you do decide to update to 4.10.14.
 matchingRules:
-- type: PromQL
-  promql:
-    promql: |
-      topk(1,
-        label_replace(group(cluster_version{type="initial",version=~"4[.][0-9][.].*"}),"born_by_4_9", "yes, so possibly touched a hyphen-hashing version", "", "")
-        or
-        label_replace(0 * group(cluster_version{type="initial",version!~"4[.][0-9][.].*"}),"born_by_4_9", "no, so will not have touched a hyphen-hashing version", "", "")
-      )
+- type: type: Always
```

Comment:
[validate-blocked-edges][1]:

```
yaml.scanner.ScannerError: mapping values are not allowed here
  in "blocked-edges/4.10.14-release-data-with-hyphen-prefix.yaml", line 8, column 13
```

I expect it wants `type: Always` and not `type: type: Always`.



[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati-graph-data/2968/pull-ci-openshift-cincinnati-graph-data-master-validate-blocked-edges/1611415573508395008#1:build-log.txt%3A53